scripts/release.mjs

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env zx
+import { argv, cwd } from 'process'
+
+await $`set -e`
```

Comment:
Do we need to `set -e` in zx? normally its just for error handling but i think since we are using js i think it will exit non-zero 0 on errors. We can run a quick test

Reply:
I tested throwing `exit 33` with and without `set -e`, and the results appeared identical. I decided to include it for discussion in case someone knows definitively if it's needed or not.

Reply:
a good test here would be a `throw new Error('test')` somewhere in the middle of the script

Reply:
Here are the various outputs for the various combinations in orange:

![Screen Shot 2022-02-02 at 10 04 48 AM](https://user-images.githubusercontent.com/20399044/152202383-b021898d-bb8d-49ad-a8fd-6e4323149b89.png)

Reply:
I went ahead and removed `set -e`. It seems unnecessary.